text/0000-tiered-browser-support.md

```diff
@@ -0,0 +1,289 @@
+Google sheet][all-data] for the visits to crates.io in the last month. I chose
+to use just crates.io because on initial analysis, the top 90% of visits to
+rust-lang.org were less varied than the top 90% of visits to crates.io.
+
+[all-data]: https://docs.google.com/spreadsheets/d/1qgszm-_-Tn8FLi2v3vicuvyct3Grzz74JWcroILRq8s/edit?usp=sharing
+
+This data does not include those users who block Google Analytics.
+
+This is the top 80% aggregated by browser and major browser version:
+
+| Browser         | Browser Version | Sessions | % of sessions | Cumulative % |
+|-----------------|-----------------|----------|---------------|--------------|
+| Chrome          | 57              | 18,040   | 34.85         | 34.85        |
+| Firefox         | 52              | 8,136    | 15.72         | 50.56        |
+| Chrome          | 56              | 7302     | 14.11         | 64.67        |
+| Safari          | 10.1 (macos)    | 1,592    | 3.08          | 67.74        |
+| Safari          | 10 (ios)        | 1,437    | 2.78          | 70.52        |
+| Safari          | 10.0.3 (macos)  | 851      | 1.64          | 72.16        |
+| Firefox         | 53              | 767      | 1.48          | 73.65        |
+| Chrome          | 55              | 717      | 1.39          | 75.03        |
+| Firefox         | 45              | 693      | 1.34          | 76.37        |
+| UC Browser      | 11              | 530      | 1.02          | 77.40        |
+| Chrome          | 58              | 520      | 1.00          | 78.40        |
+| Safari (in-app) | (not set) (ios) | 500      | 0.97          | 79.37        |
+| Firefox         | 54              | 472      | 0.91          | 80.28        |
+
+Interesting to note: Firefox 45 is the latest
+[ESR](https://www.mozilla.org/en-US/firefox/organizations/all/) (Firefox 52
+will also be an ESR but it was just released). Firefox 52 was the current major
+version for most of this past month; I'm guessing the early adopters of 53 and
+54 are likely Mozilla employees.
+
+## What do other sites in our niche support?
+
+- [GitHub][] - Current versions of Chrome, Firefox, Safari, Edge and IE 11.
+  Best effort for Firefox ESR.
+- [Discourse][] - Chrome 32+, Firefox 27+, Safari 6.1+, IE 11+, iPad 3+, iOS
+  8+, Android 4.3+ (doesn't specify which browser on the devices, doesn't look
+  like they've updated these numbers in a while)
+
+[GitHub]: https://help.github.com/articles/supported-browsers/
+[Discourse]: https://github.com/discourse/discourse#requirements
+
+# How We Teach This
+[how-we-teach-this]: #how-we-teach-this
+
+We should call this "Rust Browser Support", and we should have the tiers listed
+on the Rust Forge in a similar way to the tiers of Rust platforms supported.
+
+We should link to the tiered browser support page from places where Rust web
+content is developed and on the [Rust FAQ][].
+
+[Rust FAQ]: https://www.rust-lang.org/en-US/faq.html
+
+# Drawbacks
```

Comment:
I do mention that under the analytics section, do you think it should be repeated here?